Swiftfin/Views/ItemView/ItemView.swift

```diff
@@ -15,8 +15,34 @@ import SwiftUI
 
 struct ItemView: View {
 
+    @EnvironmentObject
+    private var router: ItemCoordinator.Router
+
     @StateObject
     private var viewModel: ItemViewModel
+    @StateObject
+    private var deleteViewModel: DeleteItemViewModel
+
+    @State
+    private var showConfirmationDialog = false
+    @State
+    private var isPresentingEventAlert = false
+    @State
+    private var isPresentingFailedAlert = false
+    @State
+    private var error: JellyfinAPIError?
+
+    private let permDelete = StoredValues[.User.enableItemDeletion]
```

Comment:
Used the `StoredValue` wrappers instead:

```swift
@StoredValue(.User.enableItemDeletion)
private var enableItemDeletion: Bool
```

Reply:
Potentially dumb question: Why is this the format for `StoredValue` but not for `Defaults' when the value is being used as a constant? Should `StoredValue` just always be used with the wrapper?

Reply:
Using them as constants isn't necessarily incorrect, however they just wouldn't respond to the updates if an item view was open in another tab and the setting was changed. It is also just consistent in their usage and not tied to the lifetime of the view.

In the places that we use the `Defaults[...]/StoredValues[...]` subscripts, those are in functions that will lookup the value every time they are run and will effectively respond to new changes.